__tests__/app.test.js

```diff
@@ -49,3 +51,14 @@ describe("tests endpoint /api/topics", () => {
       });
   });
 });
+
```

Comment:
We need a length check in the above test to avoid false passes. Also i thin k we could consolidate the 3 test above into one neat integration test